pkg/cedar/core/system_summary.go

```diff
@@ -132,15 +137,28 @@ func (c *Client) GetSystem(ctx context.Context, systemID string) (*models.CedarS
 
 type systemSummaryParamFilterOpt func(*apisystems.SystemSummaryFindListParams)
 
+type systemSummaryParamOpts struct{}
+
+// SystemSummaryParams contains methods for options to pass to system summary calls
+var SystemSummaryParams = systemSummaryParamOpts{}
```

Comment:
nice addition! Might be worth doing 
```go
func NewSystemSummaryParams() systemSummaryParamOpts {
    return systemSummaryParamOpts{}
}
```

to prevent an accidental edit of `SystemSummaryParams`

Reply:
So if we did that, would we do, for example:

```go
systemSummary, err := c.GetSystemSummary(ctx, NewSystemSummaryParams().WithDeactivatedSystems())
```

That seems a little funky, so maybe:

```go
params := NewSystemSummaryParams()
systemSummary, err := c.GetSystemSummary(ctx, params.WithDeactivatedSystems())
```

Thoughts?

Reply:
if `type systemSummaryParamOpts struct{}` exists only for the options, then I should have suggested a naming like `NewSystemSummaryOpts`

```go
opts := NewSystemSummaryOpts()
systemSummary, err := c.GetSystemSummary(ctx, opts.WithDeactivatedSystems())
```

maybe something like that? i agree though, seems a bit odd, but it makes sense why you'd silo the methods to only where they're needed instead of package globals

Reply:
Ok - scratch that. I think the way you have it is fine. The case I am worried about is probably unrealistic 🙂

Reply:
Yeah, I did it so we didn't have `cedarcore.WithDeactivatedSystems()` available on the package which doesn't make much sense outside of the the system summary file.

Reply:
Just tested, there really isn't anything to modify - sorry for the 🪿 chase

Reply:
Updated here [9f62858](https://github.com/CMSgov/easi-app/pull/2605/commits/9f6285814713c11b2e0b78a9425310adbeaf7196)

Let me know your thoughts!